app/controllers/super_admins/providers_controller.rb

```diff
@@ -1,42 +1,10 @@
 class SuperAdmins::ProvidersController < ApplicationController
   include ProviderAdminConcern
 
-  def index
-    @providers = Provider.order(name: :asc)
-  end
-
-  def new
-    @provider = Provider.new
-    render 'shared/providers/new'
-  end
-
-  def edit
-    render 'shared/providers/edit'
-  end
-
-  def update
-    if @provider.update(provider_params.except(*filtered_params))
-      redirect_to super_admins_provider_path(@provider), notice: 'Provider successfully updated'
-    else
-      render 'shared/providers/edit'
-    end
-  end
-
-  def create
-    @provider = Provider.new(provider_params)
-    if @provider.save
-      redirect_to super_admins_root_path, notice: 'Provider successfully created'
-    else
-      render 'shared/providers/new'
-    end
-  end
+  def index; end
```

Comment:
as discussed this whole file should go to and, if necessary, a new basic home page for superadmins created, i'd say

Reply:
👍